batch.rst

```diff
@@ -0,0 +1,156 @@
+.. _batch:
+
+==========
+Batch Jobs
+==========
+
+Flux supports batch jobs. Users can submit batch scripts
+into Flux, which are queued up and then run when they can be
+allocated to the requested resources. Flux, however, strays away
+from the traditional approach in that batch jobs are
+supported not only at the multi-user system level
+but also at the nested single-user levels.
```

Comment:
I wonder if we should introduce what we mean by a "batch job" in this opening section.

By that I mean something like (perhaps needs more wordsmithing)

> In Flux, a batch job is a script submitted along with a request for resources. When resources become available, the script is run as the *initial program* (alternately: "with access to the resources") of a new single-user instance of Flux, with the batch user as *instance owner*. Thus, in contrast to a traditional batch scheduler, a Flux batch job has access to fully featured Flux instance, and may not only serially execute work on allocated resources, but may load custom modules or customize parameters of existing modules, monitor and update status of allocated resources, and even submit more batch jobs to run further sub-instances of Flux.

Reply:
Great suggestion! I will add this or similar.